### DISCUSSION TITLE
Support for Umami for Analytics

### DISCUSSION THREAD

**Discussion by pnwlover** [hugo-fixit/FixIt#486] [General]
Quick question, I read from the docs that FixIt only supports Google Analytics & Fathom... 

Is it easy to configure to use [Umami](https://umami.is) instead? Lots of Hugo users have posted on the Internet how they switched from Google Analytics to Umami due to GPDR restrictions... 

Am wondering why only Google Analytics & Fathom are supported if the base framework, Hugo, has lot more integration compatibilities with 3rd party external services for analytics?

Thank you for taking the time to read this.

**Comment by Lruihao** [comment]
This is a good idea, but I don't have much time left to develop it. If anyone is willing to develop it, that would be great.

**Reply by Lruihao** [reply]
Supported since https://github.com/hugo-fixit/FixIt/commit/fb80b8b3dcb3c8129f4cd80dfdbb61c0fc041979
